### PR TITLE
Use npm for building and linting

### PR DESCRIPTION
**Pull request by MikkCZ**

### CODE REVIEW

**Comment by flodolo** [comment]
I'm starting to get a bit lost in the relations between PRs. I assume #94 would still be useful in this scenario?

**Comment by MikkCZ** [comment]
Yes. I can probably make it a single PR.

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ If you want content to appear on GitBook, it needs to be listed in [docs/SUMMARY
 
 A version with improved readibility and search is built automatically via Travis using GitBook. Updates are pushed to the `gh-pages` branch and available via GitHub Pages [here](https://mozilla-l10n.github.io/styleguides).
 
-If you want to build the GitBook locally, [install GitBook](https://toolchain.gitbook.com/setup.html) and in the root of this repository run the following commands.
+If you want to build and preview the GitBook locally, [install Npm](https://www.npmjs.com/get-npm) and in the root of this repository run the following commands.
```
Comment (flodolo) [inline README.md]: nit: I think the name is `npm`, lowercase

For the second part:

>…and run the following commands from the root of the repository:

**Comment by MikkCZ** [comment]
Seems I have some some old npm on the PATH, i need to fix it.

File: README.md [hunk 2]
```diff
@@ -6,17 +6,22 @@ If you’re reading these pages on GitHub, a version with improved readability a
 
 ## Updating the documentation
 
-Simply open a pull request adding the new file or updating an existing document. Make sure to follow [these style guidelines](https://github.com/mozilla-l10n/documentation/blob/master/misc/documentation_styleguide.md), and note that all pull requests need to be reviewed before merging. The instructions for creating new community style guides are part of the [guidelines for community style guides](docs/guidelines/styleguide_guidelines.md).
+Simply open a pull request adding the new file or updating an existing document. Make sure to follow [these style guidelines](https://github.com/mozilla-l10n/documentation/blob/master/misc/documentation_styleguide.md), and note that all pull requests need to be reviewed before merging. The instructions for creating new community style guides are part of the [guidelines for community style guides](docs/guidelines/styleguide_guidelines.md). Please also run the following commands to ensure the files are well-formated.
```
Comment (flodolo) [inline README.md]: typo: formated

I would use "correctly formatted".

**Pull request merged**